docs/docs/integrations/cloud-integrations/atlan.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
I recommend putting this page in the /integrations directory instead of /cloud-integrations -- you would need to create an index.md file in /cloud-integrations in order to get this page to show up, which doesn't seem worth it for one doc.